src/components/SettingsGrid/index.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import SettingsGrid from './SettingsGrid'
+import SettingsGridCard from './SettingsGridCard'
+
+export default SettingsGrid
+export { SettingsGridCard }
```

Comment:
we shouldn't export nested components, because if they could be re-used in another place (not in their parent component) they should be moved in `src/components`